src/styles/pages/_live.scss

```diff
@@ -253,6 +253,7 @@
 .w-event-carousel {
   display: flex;
   overflow-x: scroll;
+  padding-bottom: 12px;
```

Comment:
on smaller screens this pushes the scroll bar closer to the tabs and away from the carousel, was that the intention? Or were you trying to create more separation between the carousel itself and the tabs?

Reply:
I've changed this to margin, I want to create some spacing between carousel and tabs. So the scroll bar now is close to the carousel.